packages/reactive-element/src/decorators/property.ts

```diff
@@ -146,10 +146,8 @@ export const standardProperty = <C extends Interface<ReactiveElement>, V>(
         this.requestUpdate(name, oldValue, options);
       },
       init(this: ReactiveElement, v: V): V {
-        if (v !== undefined) {
-          this._$changeProperty(name, undefined, options);
-        }
-        return v;
+        this._$changeProperty(name, undefined, options, true);
+        return v ?? (options.defaultValue as V);
```

Comment:
What do you think about a dev mode warning that they have both a default value and an initializer?

Reply:
I think it's not really possible to detect for all cases, and having a warning that sometimes shows just seems confusing and hard to maintain.

For example, decorator v. not, accessor v. field, instance shadowing, etc.

Reply:
I hope in the future we can move to standard decorators only and this will be the place we warn. I guess not that many people would use it now. We can try to add a lint warning somewhere.